pkg/dev/Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: tag push
+
+BASE=alpine:edge
+IMAGE=dev
+
+default: push
+
+hash: Dockerfile etc/ssh/sshd_config usr/bin/ssh.sh etc/motd
+	DOCKER_CONTENT_TRUST=1 docker pull $(BASE)
+	tar cf - $^ | docker build --no-cache -t $(IMAGE):build -
+	docker run --rm $(IMAGE):build sh -c "cat $^ /lib/apk/db/installed | sha1sum" | sed 's/ .*//' > $@
+
```

Comment:
This is the old style way of building. For new packages we should use multi-stage builds. We already converted a few over to it